assets/js/util.ts

```diff
@@ -84,7 +84,7 @@ export const getModeFromAffectedList = (affectedList: string[]) => {
       "green-c",
       "green-d",
       "green-e",
-    ].filter((value) => affectedList.includes(value))
+    ].filter((value) => affectedList.includes(value)).length
```

Comment:
A little more precise than filtering (since the filtered list isn't getting saved for later use)

```suggestion
    ].some(line => affectedList.includes(line))
```